app/helpers/ops_helper.rb

```diff
@@ -7,7 +7,7 @@ module OpsHelper
   include ZonesHelper
   include SettingsTabHelper
   include SettingsLabelTagMappingHelper
-
+  include SettingsScheduleHelper
```

Comment:
Best to have a newline before method definitions

```suggestion
  include SettingsScheduleHelper
  
```